cmake/Dependencies.cmake

```diff
@@ -96,8 +96,20 @@ if(BUILD_TEST)
     list(APPEND Caffe2_DEPENDENCY_LIBS ${Benchmark_LIBRARIES})
     include_directories(SYSTEM ${Benchmark_INCLUDE_DIRS})
   else()
+    # build as static library and disable install()
+    set(CAFFE2_OLD_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
+    set(BUILD_SHARED_LIBS OFF)
+    macro (install)  # noop
+    endmacro ()
```

Comment:
Out of curiosity, would this mean that all other install() commands will be no-ops as well? Or do we recover it?